packages/jmix-front-generator/src/init.ts

```diff
@@ -81,15 +81,20 @@ function collectGenerators(generatorsDir: string, genFileName?: string): Generat
       const options = generatorExports.options;
       const params = generatorExports.params;
       const description = generatorExports.description;
```

Comment:
It looks like the commit contains code related to different ticket (sorting of generators). Please remove the unrelated code.